packages/fxa-payments-server/src/lib/PaymentProvider.ts

```diff
@@ -2,16 +2,62 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-export type ProviderType = 'paypal' | 'stripe' | 'not_chosen';
```

Comment:
I thought 'ProviderType' sounded weird in some context so I renamed it.  Apologies to Lesley.